unix_integration/src/resolver.rs

```diff
@@ -1111,6 +1115,10 @@ where
                 // Can continue!
                 auth_session
             }
+            (auth_session, PamAuthResponse::MFACode { .. }) => {
+                // Can continue!
```

Comment:
How can this continue?

Reply:
Oh sorry, I read it wrong, I think I opened github wrong and thought this was the offline path.  Sorry!